projects/ngx-location-picker/src/lib/components/ngx-location-picker.component.ts

```diff
@@ -0,0 +1,634 @@
+        searchValue,
+        this.baseUrl,
+        this.locationsLimit,
+        this.locationLayers,
+        this.prioritizeLayer,
+        this.sortBy
+      ).subscribe((response: LocationModel[] | AddressModel[] | CoordinateModel[]) => {
+        this.foundLocations = response;
+      }, (error) => {
+        console.log(error);
+      }, () => {
+        this.searching = false;
+      });
+    }
+  }
+
+  /**
+   * When a location is selected from the list.
+   *
+   * @since 4.0.0
+   */
+  onLocationSelect($event: any) {
+    this.didSearch = false;
+
+    if (this.showMap) {
+      this.removeGeometry();
+
+      if ($event.address && $event.address.addressPosition && $event.address.addressPosition.wgs84) {
+        const coords: Array<number> = [$event.address.addressPosition.wgs84.lat, $event.address.addressPosition.wgs84.lng];
+        this.addMapMarker(coords);
+      } else if ($event.addressPosition && $event.addressPosition.wgs84) {
+        const coords: Array<number> = [$event.addressPosition.wgs84.lat, $event.addressPosition.wgs84.lng];
+        this.addMapMarker(coords);
+      } else if ($event.position) {
+        if ($event.position.wgs84) {
+          const coords: Array<number> = [$event.position.wgs84.lat, $event.position.wgs84.lng];
+          this.addMapMarker(coords);
+        } else if ($event.position.geometry) {
+          const geoJson = {
+            type: 'Feature',
+            properties: {
+              name: $event.label,
+            },
+            geometry: {
+              type: $event.position.geometryShape,
+              coordinates: $event.position.geometry
+            }
+          };
+
+          this.selectedLocationGeometry = this.leafletMap.addGeoJSON(geoJson, {});
+        }
+      } else if ($event.location && $event.location.position && $event.location.position.geometry) {
+        const geoJson = {
+          type: 'Feature',
+          properties: {
+            name: $event.label,
+          },
+          geometry: {
+            type: $event.location.position.geometryShape,
+            coordinates: $event.location.position.geometry
+          }
+        };
+
+        this.selectedLocationGeometry = this.leafletMap.addGeoJSON(geoJson, {});
+      } else {
+        this.setNotification({
+          status: 'm-alert--danger',
+          text: 'Locatie kan niet op de map getoond worden.',
+          icon: 'fa-exclamation-triangle'
+        });
+      }
+    }
+
+    this.writeValue($event);
+  }
+
+  /**
+   * Implements keyboard and mouse commands
+   *
+   * @since 4.0.0
+   */
+  @HostListener('window:wheel', ['$event'])
+  @HostListener('window:keydown', ['$event'])
+  onKeyCommand(event) {
+    /**
+     * zoom in/out using ctrl + scroll to zoom in or out.
+     * Show notification if only scroll is used.
+     */
+    if (event.type === 'wheel' && this.cursorOnLeaflet) {
+      if (event.ctrlKey || event.metaKey) {
```

Comment:
This conflicts in my browser (chrome on linux) with the browser's builtin ctrl-+ zoom feature. Maybe the shift key would be a better option?